Resources/doc/annotations/converters.rst

```diff
@@ -177,6 +177,40 @@ to add joins to the query), you can add the ``repository_method`` option::
     {
     }
 
+The specified repository method will be called with the criteria in an ``array``
+as parameter. This is a good fit with Doctrine's ``findBy`` and ``findOneBy``
+methods.
+
+There are cases where you want to you use your own repository method and you
+want to map the criteria to the method signature. This is possible when you set
+the ``map_method_signature`` option to true. The default is false.
+
+    /**
+     * @Route("/user/{first_name}/{last_name}")
+     * @ParamConverter("user", class="CtorsBlogBundle:User", options={
+     *    "repository_method" = "findByFullName",
+     *    "mapping": {"first_name": "firstName", "last_name": "lastName"},
+     *    "map_method_signature" = true
+     * })
+     */
+    public function showAction(User $user)
+    {
+    }
+
+    class UserRepository
+    {
+        public function findByFullName($firstName, $lastName)
+        {
+            ...
+        }
+    }
+
+.. tip::
+
+   With de default implementation, the ``firstName`` and ``lastName``
```

Comment:
I would have said: `When map_method_signature is true, ...` because `default implementation` does not refer to anything.